### PR TITLE
docs(iac-generation): update aws iac tf command help

### PR DESCRIPTION
**Pull request by ipcrm**
## Summary

Fix aws tf generate command docs strings

## How did you test this change?

N/A

## Issue

N/A

### CODE REVIEW

File: cli/cmd/generate.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var (
 	// aws command is used to generate TF code for aws
 	generateAwsTfCommand = &cobra.Command{
 		Use:   "aws",
-		Short: "Genereate terraform code for deploying into a new AWS environment",
+		Short: "Generate and optionally execute terraform code for deploying into a new AWS environment",
```
Comment (afiune) [inline cli/cmd/generate.go:57]: ugh, my gut tells me that this is no longer a `Short` description 😂

Comment (ipcrm) [reply cli/cmd/generate.go:57]: haha, i mean by comparison to the long description it's short.

**Comment by afiune** [comment]
Check windows!

File: cli/cmd/generate.go (hunk 1 above)
Comment (rmoles) [inline cli/cmd/generate.go:57]:
```suggestion
		Short: "Generate and optionally execute Terraform, to integrate Lacework to an AWS environment",
```

**Comment by ipcrm** [comment]
@afiune @rmoles How about this?

https://github.com/lacework/go-sdk/commit/641e167bfe12e0041d2650c3cedb56a57a57d1ca#diff-87d86d87a5f4a9c54027d472c0653c27d2afd666964ac87d8cac7b7e32b70bc6R57

File: cli/cmd/generate.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ var (
 	// aws command is used to generate TF code for aws
 	generateAwsTfCommand = &cobra.Command{
 		Use:   "aws",
-		Short: "Genereate terraform code for deploying into a new AWS environment",
+		Short: "Generate and/or execute Terraform code for AWS deployment",
```
Comment (rmoles) [inline cli/cmd/generate.go:57]: Would be good to get input from others but I prefer this to the original. Although I would personally replace `deployment` with `integration`

Comment (ipcrm) [reply cli/cmd/generate.go]: done.

File: cli/cmd/generate.go [hunk 3]
```diff
@@ -68,7 +68,7 @@ By default, this command will function interactively, prompting for the required
 	* The command will prompt with the outcome of the plan and allow to view more details or continue with Terraform apply
 	* If confirmed, Terraform apply will be run, completing the setup of the cloud account
 
-This command can also be run in noninteractive mode however, only generation is supported at this time.  See help output for more details on supplying required values for generation.
+This command can also be run in noninteractive mode.  See help output for more details on supplying required values for generation.
```
Comment (rmoles) [inline cli/cmd/generate.go:71]: Is there a double space after `mode.`?

Comment (ipcrm) [reply cli/cmd/generate.go]: ugh.

**Review by rmoles — APPROVED** [review]
LGTM

**Pull request merged**